vnext/DeforkingPatches/ReactCommon/turbomodule/core/TurboModuleBinding.cpp

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include "TurboModuleBinding.h"
+
+#include <stdexcept>
+#include <string>
+
+#include <ReactCommon/LongLivedObject.h>
+#include <cxxreact/SystraceSection.h>
+
+using namespace facebook;
+
+namespace facebook {
+namespace react {
+
+/**
+ * Public API to install the TurboModule system.
+ */
+TurboModuleBinding::TurboModuleBinding(
+    const TurboModuleProviderFunctionType &moduleProvider)
+    : moduleProvider_(moduleProvider) {}
+
+void TurboModuleBinding::install(
+    jsi::Runtime &runtime,
+    std::shared_ptr<TurboModuleBinding> binding) {
+  runtime.global().setProperty(
+      runtime,
```

Comment:
this indentation is giving me a headache... :P
Is this really what we've landed on for style?
makes it hard to see what is function signature versus function code

plus the file is way too narrow. I had bumped it up to 120